### ISSUE TITLE
Changing Number of Stops Returned on Landing

### ISSUE THREAD

**Issue by qwo** [Code4HR/hrt-bus-finder#103]
Currently the App returns 2 stops, different from 6 stops from original launch.

It is recommended for users to use the stop finder to locate their actual stops.

Suggestion: a load more button to add additional stop calls - up to 6 similar to previous app behavior? 

@wbprice @bschoenfeld

**Comment by wbprice** [comment]
@bschoenfeld how did we arrive at six as the number of stops that should be displayed? How many of those are useful beyond the first couple?

**Comment by qwo** [comment]
for a use case: I'm assuming its geocentrically equal, the closest 2 stops may serve other lines or are not my most convenient stops. 
For my case, the buses that are the top two, "hampton and graydon" and 'redgate and colley' are technically closer but I typically utilize  the 'children way and redgate' stop.
This morning I could not utilize it to find the stop I was looking for and nearly missed my bus. 
*\* Now i know however I can use the stop finder to find the one I was looking for **
My attention span with the app and most general users is 1-2 minutes before they bounce.
Before launch, getting 6 stops was sufficient to cover all the ones I was interested in 90% of the time. I'm worried now the 2 stops at glance, are insufficient as much for me as much as the other users. 

Abstracting this, the 'two stops based on distance' issue I was on campus,and could not at a glance notice the typical stop I look for to return home because there are two nearer stops for different lines.

My case is "i am further away from the stops and would like to know how far the buses are to plan"
whereas I think you're thinking, "im venturing to the stops, but i'd like to know how close the bus is"(50% of the walk journied) 

TLDR: So @wbprice to me at least 4-6 is useful but i agree they may not be useful for all users past 2-4.

**Comment by bschoenfeld** [comment]
6 was pretty arbitrary, but my reasoning was that most stops actually have 2 stops, one on each side of the road, so 6 stops is really the 3 closest stops. And I thought 3 was a good number. I've ramped it back up to 6 for now.
